index.js

```diff
@@ -124,16 +125,15 @@ module.exports = {
   rules: allRules,
   configs: {
     recommended: {
-      plugins: [
-        'react',
-      ],
+      plugins: ['react'],
       parserOptions: {
         ecmaFeatures: {
           jsx: true,
         },
       },
       rules: {
         'react/display-name': 2,
+        'react/jsx-inline-conditional': 2,
```

Comment:
Because this is helps identify+fix a certain class of rendering bugs and isn't strictly stylistic, I enabled this by default. Let me know if I should change this!

Reply:
Adding a rule to the recommended config is a breaking change, so we'll basically never be doing that. Please remove it.